app/routes/users/components/UserProfile.css

```diff
@@ -82,3 +82,12 @@
   font-size: var(--font-size-sm);
   color: var(--secondary-font-color);
 }
+
+.settingsIcon {
+  transform: rotate(0deg);
```

Comment:
Is this needed? What it do?

Reply:
I added it when I had some issues with the transition being wonky. Can remove?

Reply:
It doesn’t seem to be needed, but css transition works in mysterious ways sometimes🤷🏼‍♂️